src/OpenTelemetry.AutoInstrumentation.Native/cor_profiler.h

```diff
@@ -52,8 +52,8 @@ class CorProfiler : public CorProfilerBase
     // CallTarget Members
     //
     std::shared_ptr<RejitHandler> rejit_handler = nullptr;
-    bool enable_by_ref_instrumentation = false;
-    bool enable_calltarget_state_by_ref = false;
+    bool enable_by_ref_instrumentation = true;
```

Comment:
Same here.

Reply:
Added in bc10d076daf14543982643b5269c9566b811d6ee